.drone.yml

```diff
@@ -19,6 +19,8 @@ steps:
 - name: lint
   pull: always
   image: polygonio/go-linter:v1.13
+  environment:
+    ADDITIONAL_ARGS: --new-from-rev 86548ee1cf5f1478bdd5ee285a2dd0d6f8406c76
```

Comment:
What does this do exactly? I've seen it several places but don't know it's reason.

Reply:
This tells the linter to only report lint issues that were introduced after the specified git revision. In this case the hash is the current HEAD of master.

This is really convenient because I want to lint the code I'm about to add here, but don't want to go back and fix all the existing lint errors from before we had the linter. Before we were linting and just threw in a bunch of file-level `nolint`s which would hide new lint errors as well